src/ImageSharp/Formats/Jpg/Utils/MutableSpanExtensions.cs

```diff
@@ -0,0 +1,164 @@
+﻿// <copyright file="MutableSpanExtensions.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Formats.Jpg
+{
+    using System.Numerics;
+    using System.Runtime.CompilerServices;
+
+    /// <summary>
+    /// MutableSpan Extensions
+    /// </summary>
+    internal static class MutableSpanExtensions
```

Comment:
The methods returning `MutableSpan<T>` are not "spanny", need to refactor them. (Next time, if possible.)